Exiled.API/Features/Core/Generic/StaticActor.cs

```diff
@@ -0,0 +1,169 @@
+                NotifyInstanceRepeated();
+                return;
+            }
+
+            if (!IsInitialized)
+            {
+                Log.Info($"Start() StaticActor with type {GetType().Name} in the Actor {Name}");
+                PostInitialize_Static();
+                IsInitialized = true;
+            }
+        }
+
+        /// <inheritdoc/>
+        protected override void OnBeginPlay()
+        {
+            base.OnBeginPlay();
+
+            if (IsStarted)
+                return;
+
+            BeginPlay_Static();
+            IsStarted = true;
+        }
+
+        /// <inheritdoc/>
+        protected override void OnEndPlay()
+        {
+            if (this != instance)
+                return;
+
+            IsDestroyed = true;
+            EndPlay_Static();
+        }
+
+        /// <summary>
+        /// Flushes the current actor.
+        /// </summary>
+        protected virtual void Flush() => Destroy();
+
+        /// <summary>
+        /// Fired on <see cref="PostInitialize()"/>.
+        /// </summary>
+        /// <remarks>
+        /// This method will only be called once even if multiple instances of the <see cref="StaticActor{T}"/> component exist in the scene.
+        /// <br>You can override this method in derived classes to customize the initialization of the component.</br>
+        /// </remarks>
+        protected virtual void PostInitialize_Static()
+        {
+        }
+
+        /// <summary>
+        /// Fired on <see cref="OnBeginPlay()"/>.
+        /// </summary>
+        /// <remarks>
+        /// This method will only be called once even if multiple instances of the <see cref="StaticActor{T}"/> component exist in the scene.
+        /// <br>You can override this method in derived classes to customize the initialization of the component.</br>
+        /// </remarks>
+        protected virtual void BeginPlay_Static()
+        {
+        }
+
+        /// <summary>
+        /// Fired on <see cref="OnEndPlay()"/>.
+        /// </summary>
+        /// <remarks>
+        /// This method will only be called once even if multiple instances of the <see cref="StaticActor{T}"/> component exist in the scene.
+        /// <br>You can override this method in derived classes to customize the initialization of the component.</br>
+        /// </remarks>
+        protected virtual void EndPlay_Static()
+        {
+        }
+
+        /// <summary>
+        /// If a duplicated instance of a <see cref="StaticActor{T}"/> component is loaded into the scene this method will be called instead of <see cref="PostInitialize_Static()"/>.
+        /// <br>That way you can customize what to do with repeated instances.</br>
+        /// </summary>
+        /// <remarks>
+        /// The default approach is delete the duplicated component.
+        /// </remarks>
+        protected virtual void NotifyInstanceRepeated() => Destroy(GetComponent<T>());
```

Comment:
Destroy return a bool right ? So can we make it exiting value bool instead of void ?

Reply:
Are you drunk

Reply:
![immagine](https://github.com/Exiled-Team/EXILED/assets/60253860/d4277210-d9d4-41ec-ab67-705adddc82a6)

Reply:
> ![immagine](https://github.com/Exiled-Team/EXILED/assets/60253860/d4277210-d9d4-41ec-ab67-705adddc82a6)
> 

I am on phone it's why it's have a "?" I was not able to check